### PR TITLE
GA Static checks sans db fixtures and EnvUtils setup

### PR DESCRIPTION
**Pull request by netsettler**
* Implements `USE_SAMPLE_ENVUTILS` to cause `EnvUtils` to be initialized from the sample (Acme) configuration.
* Uses `USE_SAMPLE_ENVUTILS` and (from Snovault) `NO_SERVER_FIXTURES` in the GA Static Checks script to not have to put in complicated credentials and setup.

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -30,39 +30,48 @@ def pytest_configure():
...
-PRINT("Configuring environment variables...")
+USE_SAMPLE_ENVUTILS = environ_bool("USE_SAMPLE_ENVUTILS")
 
-my_selected_account = os.environ.get("ACCOUNT_NUMBER")
+if USE_SAMPLE_ENVUTILS:
 
-# TODO: Maybe make this test programmable in env_utils sometime. -kmp 21-Jul-2022
-desired_env = 'cgap-devtest'
+    PRINT(f"EnvUtils using sample configuration template.")
+    EnvUtils.set_declared_data(EnvUtils.SAMPLE_TEMPLATE_FOR_CGAP_TESTING)
```
Comment (netsettler) [inline conftest.py:39]: Really this just does the simple part in the `then` of the `if USE_SAMPLE_ENVUTILS` or else the part in the `else` is the part that was there before, just indented. It looks far worse here than what actually happened.

File: pyproject.toml [hunk 2]
```diff
@@ -45,7 +45,7 @@ chardet = "3.0.4"
 codeguru-profiler-agent = "^1.2.4"
 colorama = "0.3.3"
 dcicpyvcf = "1.0.0"
-dcicsnovault = "^6.0.7"
+dcicsnovault = "6.0.7.2b0"  # "^6.0.7"
```
Comment (netsettler) [inline pyproject.toml:48]: There's a PR for this change, which implements `NO_SERVER_FIXTURES` environment variable.

File: .github/workflows/static-checks.yml [hunk 3]
```diff
@@ -28,32 +28,14 @@ jobs:
...
           poetry install
       - name: Static Checks
         env:
-          S3_ENCRYPT_KEY: ${{ secrets.S3_ENCRYPT_KEY }}
-          # This variable should not get used, but it's here for separation "just in case"
-          TEST_JOB_ID: cgap-static-test-${{ github.run_number }}-
-          GLOBAL_ENV_BUCKET: ${{ secrets.GLOBAL_ENV_BUCKET }}
-          ACCOUNT_NUMBER: $ {{ secrets.ACCOUNT_NUMBER }}
+          NO_SERVER_FIXTURES: TRUE
+          USE_SAMPLE_ENVUTILS: TRUE
```
Comment (netsettler) [inline .github/workflows/static-checks.yml:38]: These two environment variables allow getting rid of all the AWS and Postgres setup and various credentails.

File: conftest.py [hunk 4]
```diff
@@ -30,39 +30,48 @@ def pytest_configure():
...
+
+    my_selected_account = os.environ.get("ACCOUNT_NUMBER")
+
+    # TODO: Maybe make this test programmable in env_utils sometime. -kmp 21-Jul-2022
+    desired_env = 'cgap-devtest'
+
+    my_selected_env = os.environ.get("ENV_NAME")
+
+    if not my_selected_account or my_selected_account == "643366669028":
+        PRINT("The legacy account can no longer be used for testing cgap-portal.")
```
Comment (willronchetti) [inline conftest.py:53]: You might want to state the selected account number in this message, if there is one

**Review by willronchetti — APPROVED** [review]

**Comment by netsettler** [comment]
This passes tests on my machine.
```
=== 395 passed, 28 skipped in 1451.99 seconds ===
```

File: conftest.py (hunk 4 above)
Comment (netsettler) [reply conftest.py:53]: Oops. I missed seeing this before merging, but I note that it's saying that you're using the legacy account, which is why I thought it wasn't needed. Sometimes it's explicit (by using that account number) and sometimes implicit (by using no account number), and it _could_ say that. Another PR, I guess.

**Pull request merged**